### PR TITLE
Neftet Mob Aggro update

### PR DESCRIPTION
**Pull request by DridsGhost**
Bumped up the visual awareness setting on mobs  in dungeons and the overworld. Beefed up Burning Sands golem, wasps and a few other mobs. Clean up some weenie data.

### CODE REVIEW

File: Database/Patches/2011-05-LostCityOfNeftet/9 WeenieDefaults/Creature/Armoredillo/44015 Guardian Armoredillo.sql [hunk 1]
```diff
@@ -8,7 +8,7 @@ VALUES (44015, 'ace44015-guardianarmoredillo', 10, '2019-02-10 00:00:00') /* Cre
 INSERT INTO `weenie_properties_int` (`object_Id`, `type`, `value`)
 VALUES (44015,   1,         16) /* ItemType - Creature */
      , (44015,   2,         17) /* CreatureType - Armoredillo */
-     , (44015,   3,         12) /* PaletteTemplate - Navy */
+     , (44015,   3,         12) /* PaletteTemplate - Navy */
```
Comment (ziang4891) [inline Database/Patches/2011-05-LostCityOfNeftet/9 WeenieDefaults/Creature/Armoredillo/44015 Guardian Armoredillo.sql:11]: Shouldn't be adding trailing whitespaces

File: Database/Patches/2011-05-LostCityOfNeftet/9 WeenieDefaults/Creature/Armoredillo/44015 Guardian Armoredillo.sql [hunk 2]
```diff
@@ -77,7 +77,7 @@ VALUES (44015,   1,   33554436) /* Setup */
      , (44015,   2,  150994972) /* MotionTable */
      , (44015,   3,  536870915) /* SoundTable */
      , (44015,   4,  805306382) /* CombatTable */
-     , (44015,   7,  268435547) /* ClothingBase */
+     , (44015,   7,  268435547) /* ClothingBase */
```
Comment (ziang4891) [inline Database/Patches/2011-05-LostCityOfNeftet/9 WeenieDefaults/Creature/Armoredillo/44015 Guardian Armoredillo.sql:80]: Shouldn't be adding trailing whitespaces

File: Database/Patches/2011-05-LostCityOfNeftet/9 WeenieDefaults/Creature/Armoredillo/44017 Tamed Armoredillo.sql [hunk 3]
```diff
@@ -8,7 +8,7 @@ VALUES (44017, 'ace44017-tamedarmoredillo', 10, '2019-02-10 00:00:00') /* Creatu
 INSERT INTO `weenie_properties_int` (`object_Id`, `type`, `value`)
 VALUES (44017,   1,         16) /* ItemType - Creature */
      , (44017,   2,         17) /* CreatureType - Armoredillo */
-     , (44017,   3,          8) /* PaletteTemplate - Green */
+     , (44017,   3,          8) /* PaletteTemplate - Green */
```
Comment (ziang4891) [inline Database/Patches/2011-05-LostCityOfNeftet/9 WeenieDefaults/Creature/Armoredillo/44017 Tamed Armoredillo.sql:11]: Shouldn't be adding trailing whitespaces

File: Database/Patches/2011-05-LostCityOfNeftet/9 WeenieDefaults/Creature/Armoredillo/44017 Tamed Armoredillo.sql [hunk 4]
```diff
@@ -19,7 +19,7 @@ VALUES (44017,   1,         16) /* ItemType - Creature */
      , (44017,  93,       1032) /* PhysicsState - ReportCollisions, Gravity */
      , (44017, 133,          2) /* ShowableOnRadar - ShowMovement */
      , (44017, 146,    1400000) /* XpOverride */
-     , (44017, 332,        120) /* LuminanceAward */;
+     , (44017, 332,        120) /* LuminanceAward */;
```
Comment (ziang4891) [inline Database/Patches/2011-05-LostCityOfNeftet/9 WeenieDefaults/Creature/Armoredillo/44017 Tamed Armoredillo.sql:22]: Shouldn't be adding trailing whitespaces

File: Database/Patches/2011-05-LostCityOfNeftet/9 WeenieDefaults/Creature/Armoredillo/44017 Tamed Armoredillo.sql [hunk 5]
```diff
@@ -77,7 +77,7 @@ VALUES (44017,   1,   33554436) /* Setup */
      , (44017,   2,  150994972) /* MotionTable */
      , (44017,   3,  536870915) /* SoundTable */
      , (44017,   4,  805306382) /* CombatTable */
-     , (44017,   7,  268435547) /* ClothingBase */
+     , (44017,   7,  268435547) /* ClothingBase */
```
Comment (ziang4891) [inline Database/Patches/2011-05-LostCityOfNeftet/9 WeenieDefaults/Creature/Armoredillo/44017 Tamed Armoredillo.sql:80]: Shouldn't be adding trailing whitespaces

File: Database/Patches/2011-05-LostCityOfNeftet/9 WeenieDefaults/Creature/Armoredillo/44017 Tamed Armoredillo.sql [hunk 6]
```diff
@@ -139,4 +139,17 @@ VALUES (44017, 9, 44240,  1, 0, 0.1, False) /* Create A'nekshay Token (44240) fo
      , (44017, 9, 44295,  1, 0, 0.02, False) /* Create Ancient Tablet of the Crystal Amulet (Level 180+) (44295) for ContainTreasure */
      , (44017, 9, 44294,  1, 0, 0.02, False) /* Create Ancient Tablet of the Crystal Sword (Level 180+) (44294) for ContainTreasure */
      , (44017, 9,     0,  0, 0, 0.92, False) /* Create nothing for ContainTreasure */;
-
+
```
Comment (ziang4891) [inline Database/Patches/2011-05-LostCityOfNeftet/9 WeenieDefaults/Creature/Armoredillo/44017 Tamed Armoredillo.sql:142]: Shouldn't be adding trailing whitespaces, and missing file newline ending

File: Database/Patches/2011-05-LostCityOfNeftet/9 WeenieDefaults/Creature/Armoredillo/44019 War Armoredillo.sql [hunk 7]
```diff
@@ -8,7 +8,7 @@ VALUES (44019, 'ace44019-tamedarmoredillo', 10, '2019-02-10 00:00:00') /* Creatu
 INSERT INTO `weenie_properties_int` (`object_Id`, `type`, `value`)
 VALUES (44019,   1,         16) /* ItemType - Creature */
      , (44019,   2,         17) /* CreatureType - Armoredillo */
-     , (44019,   3,         13) /* PaletteTemplate - Purple */
+     , (44019,   3,         13) /* PaletteTemplate - Purple */
```
Comment (ziang4891) [inline Database/Patches/2011-05-LostCityOfNeftet/9 WeenieDefaults/Creature/Armoredillo/44019 War Armoredillo.sql:11]: Shouldn't be adding trailing whitespaces

File: Database/Patches/2011-05-LostCityOfNeftet/9 WeenieDefaults/Creature/Armoredillo/44019 War Armoredillo.sql [hunk 8]
```diff
@@ -19,7 +19,7 @@ VALUES (44019,   1,         16) /* ItemType - Creature */
      , (44019,  93,       1032) /* PhysicsState - ReportCollisions, Gravity */
      , (44019, 133,          2) /* ShowableOnRadar - ShowMovement */
      , (44019, 146,    1850000) /* XpOverride */
-     , (44019, 332,        200) /* LuminanceAward */;
+     , (44019, 332,        200) /* LuminanceAward */;
```
Comment (ziang4891) [inline Database/Patches/2011-05-LostCityOfNeftet/9 WeenieDefaults/Creature/Armoredillo/44019 War Armoredillo.sql:22]: Shouldn't be adding trailing whitespaces

File: Database/Patches/2011-05-LostCityOfNeftet/9 WeenieDefaults/Creature/Armoredillo/44019 War Armoredillo.sql [hunk 9]
```diff
@@ -76,7 +76,7 @@ VALUES (44019,   1,   33554436) /* Setup */
      , (44019,   2,  150994972) /* MotionTable */
      , (44019,   3,  536870915) /* SoundTable */
      , (44019,   4,  805306382) /* CombatTable */
-     , (44019,   7,  268435547) /* ClothingBase */
+     , (44019,   7,  268435547) /* ClothingBase */
```
Comment (ziang4891) [inline Database/Patches/2011-05-LostCityOfNeftet/9 WeenieDefaults/Creature/Armoredillo/44019 War Armoredillo.sql:79]: Shouldn't be adding trailing whitespaces

File: Database/Patches/2011-05-LostCityOfNeftet/9 WeenieDefaults/Creature/Armoredillo/44019 War Armoredillo.sql [hunk 10]
```diff
@@ -126,6 +126,15 @@ SET @parent_id = LAST_INSERT_ID();
 INSERT INTO `weenie_properties_emote_action` (`emote_Id`, `order`, `type`, `delay`, `extent`, `motion`, `message`, `test_String`, `min`, `max`, `min_64`, `max_64`, `min_Dbl`, `max_Dbl`, `stat`, `display`, `amount`, `amount_64`, `hero_X_P_64`, `percent`, `spell_Id`, `wealth_Rating`, `treasure_Class`, `treasure_Type`, `p_Script`, `sound`, `destination_Type`, `weenie_Class_Id`, `stack_Size`, `palette`, `shade`, `try_To_Bond`, `obj_Cell_Id`, `origin_X`, `origin_Y`, `origin_Z`, `angles_W`, `angles_X`, `angles_Y`, `angles_Z`)
 VALUES (@parent_id,  0,   5 /* Motion */, 0, 1, 268435538 /* Twitch2 */, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL);
 
+INSERT INTO `weenie_properties_emote` (`object_Id`, `category`, `probability`, `weenie_Class_Id`, `style`, `substyle`, `quest`, `vendor_Type`, `min_Health`, `max_Health`)
```
Comment (ziang4891) [inline Database/Patches/2011-05-LostCityOfNeftet/9 WeenieDefaults/Creature/Armoredillo/44019 War Armoredillo.sql:129]: In "modern" ACE killtask parlance, the Killtask stamp should be in the string table, not in the death emote

File: Database/Patches/2011-05-LostCityOfNeftet/9 WeenieDefaults/Creature/Armoredillo/44019 War Armoredillo.sql [hunk 11]
```diff
@@ -136,9 +145,10 @@ VALUES (44019, 9, 44240,  1, 0, 0.1, False) /* Create A'nekshay Token (44240) fo
      , (44019, 9, 44295,  1, 0, 0.02, False) /* Create Ancient Tablet of the Crystal Amulet (Level 180+) (44295) for ContainTreasure */
      , (44019, 9, 44294,  1, 0, 0.02, False) /* Create Ancient Tablet of the Crystal Sword (Level 180+) (44294) for ContainTreasure */
      , (44019, 9,     0,  0, 0, 0.92, False) /* Create nothing for ContainTreasure */;
-
-
-
-
-
-
+
```
Comment (ziang4891) [inline Database/Patches/2011-05-LostCityOfNeftet/9 WeenieDefaults/Creature/Armoredillo/44019 War Armoredillo.sql:148]: Shouldn't be adding trailing whitespaces, and missing file newline ending

File: Database/Patches/2011-05-LostCityOfNeftet/9 WeenieDefaults/Creature/Golem/44032 Dust Golem.sql [hunk 12]
```diff
@@ -88,25 +88,25 @@ VALUES (44032,   1,  9815, 0, 0, 10000) /* MaxHealth */
      , (44032,   5,  5000, 0, 0, 5260) /* MaxMana */;
 
 INSERT INTO `weenie_properties_skill` (`object_Id`, `type`, `level_From_P_P`, `s_a_c`, `p_p`, `init_Level`, `resistance_At_Last_Check`, `last_Used_Time`)
```
Comment (ziang4891) [inline Database/Patches/2011-05-LostCityOfNeftet/9 WeenieDefaults/Creature/Golem/44032 Dust Golem.sql:90]: Why where the skill levels adjusted upwards, and also made unequal to this weenies counterpart?  Remember that attributes are factored into the final skill values used, at least in ACE, so if the **init_Level** is increased to much, there won't be any way for a player to counter or effectively attack the mobs

File: Database/Patches/2011-05-LostCityOfNeftet/9 WeenieDefaults/Creature/Golem/44033 Burning Sands Golem.sql [hunk 13]
```diff
@@ -102,24 +102,24 @@ VALUES (44033,   1,  9815, 0, 0, 10000) /* MaxHealth */
 INSERT INTO `weenie_properties_skill` (`object_Id`, `type`, `level_From_P_P`, `s_a_c`, `p_p`, `init_Level`, `resistance_At_Last_Check`, `last_Used_Time`)
```
Comment (ziang4891) [inline Database/Patches/2011-05-LostCityOfNeftet/9 WeenieDefaults/Creature/Golem/44033 Burning Sands Golem.sql:102]: Same comment as with Dust Golem

File: Database/Patches/2011-05-LostCityOfNeftet/9 WeenieDefaults/Creature/Golem/44034 Burning Sands Golem.sql [hunk 14]
```diff
@@ -91,10 +91,10 @@ VALUES (44034,   1,  9815, 0, 0, 10000) /* MaxHealth */
 INSERT INTO `weenie_properties_skill` (`object_Id`, `type`, `level_From_P_P`, `s_a_c`, `p_p`, `init_Level`, `resistance_At_Last_Check`, `last_Used_Time`)
```
Comment (ziang4891) [inline Database/Patches/2011-05-LostCityOfNeftet/9 WeenieDefaults/Creature/Golem/44034 Burning Sands Golem.sql:91]: Same comment as with Dust Golem

File: Database/Patches/2011-05-LostCityOfNeftet/9 WeenieDefaults/Creature/Virindi/43979 Virindi Dreamslayer.sql [hunk 15]
```diff
@@ -89,10 +89,10 @@ VALUES (43979,   1, 210, 0, 0) /* Strength */
 INSERT INTO `weenie_properties_skill` (`object_Id`, `type`, `level_From_P_P`, `s_a_c`, `p_p`, `init_Level`, `resistance_At_Last_Check`, `last_Used_Time`)
```
Comment (ziang4891) [inline Database/Patches/2011-05-LostCityOfNeftet/9 WeenieDefaults/Creature/Virindi/43979 Virindi Dreamslayer.sql:89]: Same comment as with Dust Golem

File: Database/Patches/2011-05-LostCityOfNeftet/9 WeenieDefaults/Creature/Wasp/52288 Desert Phyntos Swarm.sql [hunk 16]
```diff
@@ -84,34 +84,34 @@ INSERT INTO `weenie_properties_attribute_2nd` (`object_Id`, `type`, `init_Level`
 VALUES (52288,   1,  1750, 0, 0, 1850) /* MaxHealth */
      , (52288,   3,  1800, 0, 0, 2000) /* MaxStamina */
      , (52288,   5,  1600, 0, 0, 1760) /* MaxMana */;
-	 
+
 INSERT INTO `weenie_properties_skill` (`object_Id`, `type`, `level_From_P_P`, `s_a_c`, `p_p`, `init_Level`, `resistance_At_Last_Check`, `last_Used_Time`)
```
Comment (ziang4891) [inline Database/Patches/2011-05-LostCityOfNeftet/9 WeenieDefaults/Creature/Wasp/52288 Desert Phyntos Swarm.sql:88]: Same comment as with Dust Golem

**Comment by ziang4891** [comment]
I do believe that Neftet is one of the seven areas that should be tier 8.  The death treasure entries are also inconsistent between purposely duplicated mobs, ie. those with Lum and those without.

**Comment by DridsGhost** [comment]
> Many of the columns don't appear to be lining up, possibly because of tabbing vs spaces. The format convention is for a tab size of 4, and for my own use, I've set Notepad++ to convert all tabs to spaces to avoid any git/Github conversion issues.

I have no idea why this happened.

**Comment by DridsGhost** [comment]
> I do believe that Neftet is one of the seven areas that should be tier 8. The death treasure entries are also inconsistent between purposely duplicated mobs, ie. those with Lum and those without.

Neftet is Tier 8, OverWorld Mobs are set to 2000 and award Luminance. Indoor mobs are no Lum and T7 per EoR.

File: Database/Patches/2011-05-LostCityOfNeftet/9 WeenieDefaults/Creature/Armoredillo/44015 Guardian Armoredillo.sql [hunk 17]
```diff
@@ -139,4 +139,26 @@ VALUES (44015, 9, 44240,  1, 0, 0.1, False) /* Create A'nekshay Token (44240) fo
      , (44015, 9, 44295,  1, 0, 0.02, False) /* Create Ancient Tablet of the Crystal Amulet (Level 180+) (44295) for ContainTreasure */
      , (44015, 9, 44294,  1, 0, 0.02, False) /* Create Ancient Tablet of the Crystal Sword (Level 180+) (44294) for ContainTreasure */
      , (44015, 9,     0,  0, 0, 0.92, False) /* Create nothing for ContainTreasure */;
-
+
```
[the remark this answers is not in the document]
Comment (DridsGhost) [reply Database/Patches/2011-05-LostCityOfNeftet/9 WeenieDefaults/Creature/Armoredillo/44015 Guardian Armoredillo.sql:142]: No idea why this happened. I will look into it.

File: Database/Patches/2011-05-LostCityOfNeftet/9 WeenieDefaults/Creature/Golem/44032 Dust Golem.sql (hunk 12 above)
Comment (DridsGhost) [reply Database/Patches/2011-05-LostCityOfNeftet/9 WeenieDefaults/Creature/Golem/44032 Dust Golem.sql:90]: Players noted that the burning sands golem and dust golems on the landscape originally seemed to have their difficulties reversed. This is to address that concern.

**Comment by DridsGhost** [comment]
gitCapture
npCapture


I dont see an issue. I went through every file except maybe 2 or 3 and straightened them out.

Not sure what I am missing. But when I view and or edit inside GitHub as a test to look at the formatting, its fine to me.

**Comment by ziang4891** [comment]
What are you using to edit the SQL files?  If VS, see https://github.com/github/VisualStudio/issues/1524

File: Database/Patches/2011-05-LostCityOfNeftet/9 WeenieDefaults/Creature/Anekshay/48801 Janthef.sql [hunk 18]
```diff
@@ -105,7 +104,7 @@ VALUES (48801,  6, 0, 2, 0, 470, 0, 0) /* MeleeDefense        Trained */
      , (48801, 24, 0, 2, 0, 255, 0, 0) /* Run                 Trained */
      , (48801, 33, 0, 2, 0, 500, 0, 0) /* LifeMagic           Trained */
      , (48801, 34, 0, 2, 0, 480, 0, 0) /* WarMagic            Trained */
-	 , (48801, 13, 0, 3, 0, 490, 0, 0) /* UnarmedCombat       Specialized */
+     , (48801, 13, 0, 3, 0, 490, 0, 0) /* UnarmedCombat       Specialized */
```
[the remark this answers is not in the document]
Comment (DridsGhost) [reply Database/Patches/2011-05-LostCityOfNeftet/9 WeenieDefaults/Creature/Anekshay/48801 Janthef.sql]: removed

File: Database/Patches/2011-05-LostCityOfNeftet/9 WeenieDefaults/Creature/Golem/44034 Burning Sands Golem.sql [hunk 19]
```diff
@@ -16,8 +16,8 @@ VALUES (44034,   1,         16) /* ItemType - Creature */
      , (44034,  93,       1032) /* PhysicsState - ReportCollisions, Gravity */
      , (44034, 133,          2) /* ShowableOnRadar - ShowMovement */
      , (44034, 146,    4000000) /* XpOverride */
-	 , (44034, 307,         10) /* DamageRating */
-	 , (44034, 315,       9999) /* CritResistRating */;
+     , (44034, 307,         10) /* DamageRating */
+     , (44034, 315,       9999) /* CritResistRating */;
```
Comment (ziang4891) [inline Database/Patches/2011-05-LostCityOfNeftet/9 WeenieDefaults/Creature/Golem/44034 Burning Sands Golem.sql:20]: Are you seriously kidding me?  9999

Comment (DridsGhost) [reply Database/Patches/2011-05-LostCityOfNeftet/9 WeenieDefaults/Creature/Golem/44034 Burning Sands Golem.sql:20]: The following had appraise data and was used for 44034

http://ac.yotesfan.com/weenies/items/44033

**Pull request merged**